oak/asylo_rust/src/lib.rs

```diff
@@ -34,13 +32,39 @@ extern crate alloc;
 extern crate core;
 extern crate libc;
 
+extern crate rust;
+
 pub use alloc::prelude::v1::*;
 
-pub mod enclave;
-pub mod common;
+pub use rust::error as error;
+pub use rust::io as io;
+
+mod allocator;
+mod asylo;
+pub use self::asylo::*;
 
 #[global_allocator]
-static A: enclave::allocator::System = enclave::allocator::System;
+static A: allocator::System = allocator::System;
+
+// TODO: Enclave backend that provides:
+// - Allocator
+// - Threads
+// - Mutexes
+// - TODO: Channel with automatic remote attestation
+
+/// A safe function to spawn an enclave thread. At the moment, parameters cannot
+/// be passed in this function call, unlike the rust standard library. (This can
```

Comment:
nit: s/rust/Rust/

Reply:
Fixed

Reply:
Not seeing this &ndash; lost in a merge somewhere?